perform-release.sh

```diff
@@ -20,7 +20,7 @@ RELEASE_VERSION=$(getVersion)
 echo "Performing release ${RELEASE_VERSION}"
 
 echo "Deploying release ${RELEASE_VERSION}"
-./mvnw clean deploy -pl "\!osgi-support, \!osgi-support/mirror-subclass-plugin/, \!osgi-support/sdk-osgi-test-campaign/, \!osgi-support/commercetools-tests-fragment" -DskipTests -Prelease,javadoc-jdk-8u121
+./mvnw clean deploy -pl "!osgi-support, !osgi-support/mirror-subclass-plugin, !osgi-support/sdk-osgi-test-campaign, !osgi-support/commercetools-tests-fragment" -DskipTests -Prelease,javadoc-jdk-8u121
```

Comment:
@katmatt how about this small diffs here ?

Reply:
They're fine and I don't remember why we introduced the `\`.